package.json

```diff
@@ -25,18 +27,23 @@
   },
   "homepage": "https://github.com/Andarist/use-value-callbag#readme",
   "peerDependencies": {
-    "react": "^16.7.0-alpha.0"
+    "react": "^16.8 || ^17"
   },
   "devDependencies": {
     "@babel/core": "^7.1.2",
     "@babel/plugin-transform-modules-commonjs": "^7.1.0",
-    "@babel/preset-env": "^7.1.0",
+    "@babel/preset-env": "^7.13.10",
+    "@babel/preset-react": "^7.12.13",
+    "@testing-library/react-hooks": "^5.1.0",
+    "callbag": "^1.4.0",
+    "callbag-common": "^0.1.8",
     "husky": "^1.1.0",
-    "jest": "^23.6.0",
+    "jest": "^26.6.3",
     "lint-staged": "^7.3.0",
     "prettier": "^1.14.3",
-    "react": "16.7.0-alpha.0",
-    "react-dom": "16.7.0-alpha.0",
+    "react": "^17.0.1",
+    "react-dom": "^17.0.1",
+    "react-test-renderer": "^17.0.1",
```

Comment:
in the end this one ain't used, right?

Reply:
it actually is, as it is a [peer dependency for react-hooks-testing-library](https://github.com/testing-library/react-hooks-testing-library#peer-dependencies).

Reply:
gotcha 👍